README.rst

```diff
@@ -78,34 +88,31 @@ Clone the `CPython repository`_ and build it (you will be cleaning up your build
 later, though as a final step).
 
 Also make sure to build the documentation. This alleviates the need for
-everyone to build it from scratch. To build the documentation, create a venv
+sprint participants to build it from scratch. To build the documentation, create a venv
 with sphinx installed and point the Doc Makefile at the Python linked to in the
 venv.
 
 All of this can be done by doing::
 
-  # Assuming at the top of the devinabox.
+  # Assuming at the root of the devinabox directory
   python build_cpython.py
   ./cpython/python -m venv venv
   ./venv/bin/pip install sphinx
   cd cpython/Doc
   make html PYTHON=../../venv/bin/python
 
-.. _CPython repository: http://hg.python.org/cpython
+.. _CPython repository: https://github.com/python/devinabox
```

Comment:
This link target is for the CPython repo, which hasn't moved yet, rather than the devinabox repo itself